gradle.properties

```diff
@@ -0,0 +1,11 @@
+# Scan tool properties
+group=io.ballerina.scan
+scanToolVersion=1.0.0
```

Comment:
Let's start with a lower version.

Reply:
Since this isn't yet production-ready, nor will it be in the initial iteration.

Reply:
Fixed in, [1ff55d9](https://github.com/ballerina-platform/static-code-analysis-tool/pull/2/commits/1ff55d96978f6e19798757dc259d3fb5d91f32f3)